sunbird-cassandra-migration/cassandra-migration/src/main/resources/db/migration/cassandra/V1.106_cassandra.cql

```diff
@@ -0,0 +1 @@
+CREATE TABLE sunbird.user_organisation (userid text,organisationid text,addedby text,addedbyname text,approvaldate text,approvedby text,hashtagid text,id text,isapproved boolean,isdeleted boolean,isrejected boolean,orgjoindate text,orgleftdate text,position text,roles list<text>,updatedby text,updateddate text,PRIMARY KEY (userid, organisationid));
```

Comment:
can you call orgId only? If this is just primary key change, then perhaps it is easy to migrate data.

Reply:
looks I picked from 2.10 mw repo where it refers to orgId. Merging after a short call.